torch/onnx/_internal/fx/dynamo_graph_extractor.py

```diff
@@ -45,7 +45,7 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         for class_type in self._extensions:
-            pytree.SUPPORTED_NODES.pop(class_type)
+            pytree.SUPPORTED_NODES.pop(id(class_type))
```

Comment:
This suggests changing SUPPORTED_NODES is BC-breaking, you may need SUPPORTED_NODE_IDS